spec/feature_flagger/configuration_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+module FeatureFlagger
+  RSpec.describe Configuration do
+    context 'accessors methods' do
+      subject(:configuration) { described_class.new }
+
+      it { expect(configuration).to respond_to(:storage) }
+      it { expect(configuration).to respond_to(:storage=) }
+      it { expect(configuration).to respond_to(:yaml_filepath) }
+      it { expect(configuration).to respond_to(:info) }
+    end
```

Comment:
@geisonbiazus I'm with some problems to test the `yaml_filepath` and `info` to assert default values. 

If you see it https://github.com/ResultadosDigitais/feature_flagger/pull/17/files#diff-ad14e932d27d63eebd703a30be61599cR24 the default value depens on `defined?` of `Rails` const, how can I simulate this is my specs ? Using `allow` or something ?